### DISCUSSION TITLE
Delayed interaction with hover

### DISCUSSION THREAD

**Discussion by DerekLee13** [WindowTop/WindowTop-App#305] [Ideas]
I don't *think* this is an option yet, but I would love to be able do click through a PIP window but trigger Interact in the PIP window after hovering for an adjustable amount of time. 

Example: I'm working on a Word document while playing a Netflix video in PIP. Currently, if I have Interact enabled, I can pause the video by moving the pointer over the PIP window and clicking. But to be able to work on the document behind the PIP, I have to actually move the PIP window to get to the area I need to work on. Conversely, If I have click-through enabled, I can work behind the PIP window without moving it, but to pause the video (or any other interaction), I have to exit PIP/WIndowTop mode to do so. I'd love to be able to hover over the PIP window for 1 or 2 seconds to trigger Interact mode to work within the PIP, then after leaving the PIP window for another 1 or 2 seconds, goes back to click-through. 

If this is already a feature, I apologize for wasting space here. But if something like this would be amazing. Or if someone has a better idea to accomplish something similar, please share!

**Comment by gileli121** [comment]
Hi,
Thanks for the suggestion.
There is no such option, and I will consider implementing it.
I will also try to think about other ideas that may be better instead this one.
